internal/provider/helper_test.go

```diff
@@ -71,6 +73,22 @@ func testAccCheck(t *testing.T) {
 	}
 }
 
+// Helper Function that accepts two skipFunc and returns "true" if either returns true.
+func orSkipFunc(input ...SkipFunc) func() (bool, error) {
```

Comment:
```suggestion
// orSkipFunc accepts many skipFunc and returns "true" if any returns true.
func orSkipFunc(input ...SkipFunc) SkipFunc {
```

Reply:
Of _course_ I forgot to update the comment 🤦‍♂️ .